### ISSUE TITLE
Make a release

### ISSUE THREAD

**Issue by vitlav** [beanstalkd/beanstalkd#383]
Please, make a release. At least, every year.

**Comment by JensRantil** [comment]
...even better, make it on every commit through CI.

**Comment by silviucpp** [comment]
:)) I never seen any reply from the author of the project for more than 1 year :)) This project is absolutely dead

**Comment by vitlav** [comment]
will somebody take care about pull requests?

**Comment by stamster** [comment]
No one has access to handle PR's?
That's why fork is best choice here. Fork it, and start with a team.

**Comment by criscola** [comment]
@silviucpp I hope the author is not dead tough

**Comment by silviucpp** [comment]
:)) he is fine for sure . I saw activity in other repos. Most probably he
doesn’t care about project any longer

**Comment by JensRantil** [comment]
Hello! I just stepped up as an additional maintainer to get this project moving a bit again. I believe making a release is one of the highest priorities before starting to look at outstanding issues and pending pull requests.

**Comment by JensRantil** [comment]
> will somebody take care about pull requests?

@vitlav Any specific pull requests that you were thinking of?

**Comment by ysmolsky** [comment]
Closing in favor of #399
